test/unit/org/apache/cassandra/db/NativeCellTest.java

```diff
@@ -125,7 +125,7 @@ private static Cell<?> rndcell(ColumnMetadata col)
     {
         long timestamp = rand.nextLong();
         int ttl = rand.nextInt();
-        int localDeletionTime = rand.nextInt();
+        long localDeletionTime = Math.abs(rand.nextInt());
```

Comment:
Maybe this would be clearer with `Cell.deletionTimeGuavaUintToLong(rand.nextInt())`.